rebasehelper/plugins/spec_hooks/replace_old_version.py

```diff
@@ -122,7 +123,8 @@ def run(cls, spec_file: SpecFile, rebase_spec_file: SpecFile, **kwargs: Any):
             if sec_name.startswith('%changelog'):
                 continue
             for index, line in enumerate(section):
-                if index in examined_lines[sec_name] or any(line.startswith(tag) for tag in cls.IGNORED_TAGS):
+                tag_ignored = any(line.startswith(MacroHelper.expand(tag)) for tag in cls.IGNORED_TAGS)
```

Comment:
This change has no effect whatsoever. Expanding a tag name does nothing.
Also, using `MacroHelper.expand()` without `default` argument (and not handling `None` return value) is going to lead to a traceback sooner or later.